src/main/riddl/Tenant/tenant.riddl

```diff
@@ -28,129 +46,158 @@ context TenantContext is {
+        //                then "add orgs to info's list of tenant orgs"
+        //                and "Set Tenant meta data to reflect change"
+        //                and "yield OrganizationsAdded event"
+        //            }
+        //            example OrganizationsNameAlreadyInList {
+        //                when "This organizations already belongs to this tenant"
+        //                then error "Organizations already belongs to tenant"
+        //            }
+        //        }
+        //        on command RemoveOrganizations {
+        //           example Success {
+        //               then "remove organizations from info's list of tenant orgs"
+        //               and "Set Tenant meta data to reflect change"
+        //               and "yield OrganizationsRemoved event"
+        //           }
+        //           example OrganizationsNameNotInListAlready {
+        //               when "This tenant already doesn't belong to this organizations"
+        //               then error "Tenant doesn't belongs to organizations already"
+        //           }
+        //        }
+        //        // on command SwapBillingAccount {
+        //        //     example Success {
+        //        //         then "Update the BillingAccountId"
+        //        //         and "set tenant meta data to reflect change"
+        //        //         and "yield BillingAccountSwapped event"
+        //        //     }
+        //        //     example InvalidBillingAccount {
+        //        //         when "BillingAccount ID does not exist"
+        //        //         then error "BillingAccount ID does not exist"
+        //        //     }
+        //        //     example NotAuthorized {
+        //        //         when "updating user is not authorized to make changes to the tenant"
+        //        //         then error "User is not authorized to modify Tenant"
+        //        //     }
+        //        // }
+        //        // on command SwapDistributionAccount {
+        //        //     example Success {
+        //        //         then "Update the DistributionAccountId"
+        //        //         and "set tenant meta data to reflect change"
+        //        //         and "yield DistributionAccountSwapped event"
+        //        //     }
+        //        //     example InvalidBillingAccount {
+        //        //         when "DistributionAccount ID does not exist"
+        //        //         then error "DistributionAccount ID does not exist"
+        //        //     }
+        //        //     example NotAuthorized {
+        //        //         when "updating user is not authorized to make changes to the tenant"
+        //        //         then error "User is not authorized to modify Tenant"
+        //        //     }
+        //        // }
+        //        on other { then error "Command not allowed in Draft state" }
+        //    }
+        //}
```

Comment:
Because the above is double commented out, you're effectively deleting this state?

Reply:
@AlexWeinstein92 - this question is blocking the merge

Reply:
Yes, this state is non-existent after being commented out. The double comments are because there was previously commented out code in the state.

I am only not deleting it in case James decides for some reason to have Draft for Tenant.